lib/rules/declaration-block-no-duplicate-properties/index.js

```diff
@@ -117,19 +116,21 @@ const rule = (primary, secondaryOptions, context) => {
 							return;
 						}
 
-						if (ignorePrefixlessSameValues) {
-							// fails if values of consecutive, unprefixed duplicates are equal
-							if (vendor.unprefixed(value) !== vendor.unprefixed(duplicateValue)) {
-								if (context.fix) {
-									if (!important && duplicateImportant) {
-										decl.remove();
-									} else {
-										removePreviousDuplicate(decls, lowerProp);
-									}
+						if (!important && duplicateImportant) {
```

Comment:
minor

put `!important && duplicateImportant` in a const
(being reused at l145)